lib/jsonld-signatures.js

```diff
@@ -36,15 +36,25 @@ api.verify = async function verify(document, {
   suite, purpose, documentLoader, expansionMap, compactProof} = {}) {
   const result = await new ProofSet().verify(
     document, {suite, purpose, documentLoader, expansionMap, compactProof});
-  if(!documentLoader && result.error) {
-    const {error} = result;
-    if(error.name === 'jsonld.InvalidUrl') {
+  const {error} = result;
+  if(error) {
+    if(!documentLoader && error.name === 'jsonld.InvalidUrl') {
```

Comment:
Looks like in the array-of-errors case this is checking .name of an array.  I imagine that would work, but it's strange.